app/android/app/build.gradle

```diff
@@ -82,7 +82,7 @@ android {
 
 		signingConfigs {
 				release {
-						storeFile file(keystoreProperties['storeFile'])
+						storeFile file("key.jks")
```

Comment:
Why was this change committed?

Reply:
@rutajdash 
Sir, when i try to run the app with 
storeFile file(keystoreProperties['storeFile'])

The following error pops up in the terminal
![Screenshot from 2023-09-20 23-18-46](https://github.com/Monday-Morning/project-pegasus/assets/128830206/16574964-f702-4e1e-a700-42c5ef1d8889)
Launching lib/main.dart on SM E225F in debug mode...

FAILURE: Build failed with an exception.

* Where:
Build file '/home/peeyushdas/Desktop/MONDAY MORNING 2023/project-pegasus/app/android/app/build.gradle' line: 85

* What went wrong:
A problem occurred evaluating project ':app'.
> path may not be null or empty string. path='null'


And then I searched for the issue on stackoverflow, and I got this solution:-

link:- https://stackoverflow.com/questions/54457245/a-problem-occurred-evaluating-project-app-path-may-not-be-null-or-empty-st

![Screenshot from 2023-09-20 23-18-15](https://github.com/Monday-Morning/project-pegasus/assets/128830206/ae206cb9-11d8-417c-96ef-fc52d9c20c06)

Reply:
Yes, we use a keystore properties file for production builds along with credential files, which for obvious reasons is not shared to all developers.
For development, making this change is acceptable but it should not be committed but only kept on your local system.

Reply:
Ok sir.